app.rb

```diff
@@ -3,15 +3,19 @@
 require './rental'
 require './teacher'
 require './student'
+require 'json'
 
 # rubocop:disable Metrics/ClassLength
 class Library
   def initialize
     @rentals = []
     @books = []
     @people = []
+    load_data
+    load_reantals
```

Comment:
- [ ] Kindly make sure that the app works as intended even if the file you're saving/loading data into, doesn't exist. As per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_preserve_data.md#:~:text=Remember%20to%20handle%20the%20case%20when%20one%20or%20more%20of%20the%20files%20are%20missing.) requierment.